packages/edit-post/src/components/header/index.js

```diff
@@ -123,7 +120,7 @@ function Header( {
 								}
 							) }
 						>
-							<BlockContextualToolbar isFixed />
+							<BlockToolbar hideDragHandle variant="unstyled" />
```

Comment:
It's not clear from this prop what "unstyled" means. Also, shouldn't it be the default?

Reply:
Why? In this PR it will become a public API, so we have to be careful about what to expose and what the defaults are?

Reply:
> Why? In this PR it will become a public API, so we have to be careful about what to expose and what the defaults are?

Here we're removing the private `BlockContextualToolbar` and replacing it with the already public `BlockToolbar`.

It looks like the `variant` prop is all that's been added to `BlockToolbar`, and that makes sense we're now wrapping the also already public `NavigableToolbar` which already included the `variant` prop.